app/views/content/is-teaching-right-for-me/_questions.html.erb

```diff
@@ -0,0 +1,27 @@
+<div class="category__questions__container">
+
+  <section class="container category__questions">
+
+    <div class="category__question">
+      <h2>What does a teaching career look like?</h2>
```

Comment:
I'm just wondering if we have a few too many questions on this page? might it be a bit more impactful if the headings in this section were more like:

'What a career in teaching could look like'
'Make a difference'